From f425273e2f977e221284863c5205c2d56e9a7988 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 14 Mar 2014 15:52:41 +0000
Subject: [PATCH 0527/1077] MANTIS 0031531 change mstarter: configuration
 comment: how to set the delimiter in PLATFORM.EXEC

mstarter.exe
V1.1.1.9





git-svn-id: svn://localhost/SelfServiceCommon/trunk@657 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/Controller/inc/mstarter_def.h           | 10 +++++++---
 .../Massai/cpp/Controller/src/mstarter_config.cpp      |  1 +
 2 files changed, 8 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Controller/inc/mstarter_def.h b/SelfServiceCommon/Massai/cpp/Controller/inc/mstarter_def.h
index e4f68c6a..38b40809 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/inc/mstarter_def.h
+++ b/SelfServiceCommon/Massai/cpp/Controller/inc/mstarter_def.h
@@ -1,5 +1,5 @@
-#define FILE_VER_BIN     1,1,1,8
-#define FILE_VER_STR     "FileVersion", "1.1.1.8"
+#define FILE_VER_BIN     1,1,1,9
+#define FILE_VER_STR     "FileVersion", "1.1.1.9"
 
 /*!
 \file
@@ -7,12 +7,16 @@
 
 \page mstarter.exe Release Notes - mstarter.exe - process manager
 
-@version 1.1.1.8
+@version 1.1.1.9
 
 \section change1 Change history
 
 mstarter.exe
 
+\version 1.1.1.9
+\li 14.03.2014
+\li change - SteT Mantis 0031531: mstarter: configuration comment: how to set the delimiter in PLATFORM.EXEC
+
 \version 1.1.1.8
 \li 24.07.2013
 \li extension - JK Mantis 24145
diff --git a/SelfServiceCommon/Massai/cpp/Controller/src/mstarter_config.cpp b/SelfServiceCommon/Massai/cpp/Controller/src/mstarter_config.cpp
index d6346d5a..f12c7227 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/src/mstarter_config.cpp
+++ b/SelfServiceCommon/Massai/cpp/Controller/src/mstarter_config.cpp
@@ -135,6 +135,7 @@ namespace
       "List of executables depending on Hardware Devices and Software Services.\n"
       "Example: \"C:\\Programme\\Java\\jre1.7\\bin\\java.exe\" -Xrs -jar ..\\lib\\massai-platform.jar; show=yes\n"
       "  change the delimiter if ; is used in the options\n"
+      "  start the line with new delimiter followed by a space\n"
       "    Example: $ \"C:\\Programme\\Java\\jre1.7\\bin\\java.exe\" -Xrs -jar ..\\lib\\massai-platform.jar$ show=yes\n"
       "Options: (please separate all options by blank)\n"
       "  wnd = \"window name\" - specifies the window name used for process shutdown and activation\n"
-- 
2.41.0.windows.1

